src/components/data-management/ProjectsList.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Space } from 'antd';
+
+import ProjectCard from './ProjectCard';
+
+const ProjectsListContainer = (props) => {
+  const { height, filter } = props;
+
+  const projects = useSelector((state) => state.projects);
+
+  return (
+    <Space direction='vertical' style={{ width: '100%', height: height - 90 }}>
+      {
+        projects.ids.map((projectUuid) => {
+          const project = projects[projectUuid];
+
+          const matchFilter = project.name.match(filter)
+              || project.experiments.some((experimentId) => experimentId.match(filter))
+              || projectUuid.match(filter);
+
+          if (!matchFilter) return <></>;
+
+          return (
+            <ProjectCard
+              key={project.uuid}
```

Comment:
this `key` is not being used in the `ProjectCard` component -- I think you forgot to delete it. Also, make sure you add a key to each Card rendered within ProjectCard, because right now when I open the code locally, I can see a warning message about `Warning: Each child in a list should have a unique "key" prop.`

Reply:
The error is actually because of line 25 in `ProjectsListContainer`. The `Skeleton`s doesn't have keys.  I've added them.

Reply:
great!